packages/loader/container-loader/src/container.ts

```diff
@@ -1427,6 +1342,26 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this.connectionStateHandler.receivedRemoveMemberEvent(clientId);
         });
 
+        // back compat for old containers where the code details
```

Comment:
@anthony-murphy , do we actually need this?
I believe this is only needed if there is no snapshot created for old file (or someone decided to load from ops only), right?
Reason I ask - I'd really love to remove code in Contaier.load() that connects to web socket if there is no snapshot. It's is confusing too many people :) If file was never summarized, I'd consider it broken / abandoned at that moment